tests/Behat/Context/Domain/CustomerOrderCommentsContext.php

```diff
@@ -54,7 +59,10 @@ public function aCustomerTryToCommentsTheOrderWithEmptyMessage(OrderInterface $o
         /** @var ShopUserInterface $user */
         $user = $this->sharedStorage->get('user');
         try {
-            $this->sharedStorage->set('comment', new Comment($order, $user->getEmail(), '', true));
+            $comment = new Comment($order, $user->getEmail(), '', true);
```

Comment:
Since creating a new comment and setting it in the shared storage happens several times in this context, extracting this logic to a separate method would be a nice change :)

```
private function commentOrder(OrderInterface $order, string $email, string $message, bool notifyCustomer): void 
{
    $comment = new Comment($order, $email, $message, notifyCustomer);
    $comment->orderCommented();
    $this->sharedStorage->set('comment', $comment);
}
```